js/src/views/Signer/containers/RequestsPage/requestsPage.js

```diff
@@ -100,7 +103,7 @@ class RequestsPage extends Component {
           <div className={ styles.noRequestsMsg }>
             <FormattedMessage
               id='signer.requestsPage.noPending'
-              defaultMessage='There are no requests requiring your confirmation.'
+              defaultMessage='There are no transactions requiring your confirmation.'
```

Comment:
Gah, this one is tricky. I would appreciate thoughts of what to change it to.

To me, "requests" makes absolutely no sense from an economics standpoint.
When I first downloaded this app, I was initially thinking, "I have to approve people sending me money? That's silly". But now I know what it does.

How about something akin to,
"There is nothing requiring your confirmation."

Simple. Really lacks specificity, however.

BETTER YET, hide the freakin div all together. Say nothing.